main.go

```diff
@@ -0,0 +1,29 @@
+package main
```

Comment:
- Package comment
- This should be a library you suppose to import into your project, so let's rename the package and the file name if you want